core/embed/rust/src/ui/model_tr/component/pin.rs

```diff
@@ -9,6 +9,8 @@ use crate::ui::{
 use super::{theme, Button, ButtonMsg, ButtonPos};
 use heapless::String;
 
+const MAX_LENGTH: usize = 9;
```

Comment:
Why not 50?

Reply:
Just copied it from `TT`'s PIN dialogue, which also has currently 9 - 
https://github.com/trezor/trezor-firmware/blob/18c830433922690cc9b793b1ee63675eb5e4a63d/core/embed/rust/src/ui/model_tt/component/keyboard/pin.rs#L25

Reply:
#2227 increases this to 50

Reply:
Also increased it to 50 in [71740a4](https://github.com/trezor/trezor-firmware/pull/2233/commits/71740a44445495b4d36aca38074c199ce6fcffaf)